src/data/markdown/docs/30 xk6-browser/01 xk6-browser/09 Locator/click--options--.md

```diff
@@ -0,0 +1,39 @@
+---
+title: 'click([options])'
+excerpt: 'xk6-browser: locator.click method'
+---
+
+<Blockquote mod="warning">
+
+There is a known issue with this feature. See [issue #471](https://github.com/grafana/xk6-browser/issues/471) for details.
+
+</Blockquote>
+
+Mouse click on the chosen selector.
+
+| Parameter | Type   | Description                               |
+| --------- | ------ | ----------------------------------------- |
+| options   | object | See [options](#options) for more details. |
+
+### options
+
+<!-- vale off -->
+
+| Option      | Type    | Default | Description                                                                                                                                                                                                                           |
+|-------------|---------|---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| button      | string  | `left`  | The mouse button (`left`, `middle` or `right`) to use during the action.                                                                                                                                                              |
+| clickCount  | number  | `1`     | The number of times the action is performed.                                                                                                                                                                                          |
+| delay       | number  | `0`     | Milliseconds to wait between `mousedown` and `mouseup`.                                                                                                                                                                               |
+| force       | boolean | `false` | Setting this to `true` will bypass the actionability checks (`visible`, `stable`, `enabled`).                                                                                                                                         |
+| modifiers   | string  | `null`  | `Alt`, `Control`, `Meta` or `Shift` modifiers keys pressed during the action. If not specified, currently pressed modifiers are used.                                                                                                 |
```

Comment:
The `modifiers` type is an array of strings.

Maybe we should start specifying types in TypeScript notation, so `string[]` in this case? It's fine if we just leave it as `array` for now, though.

Reply:
Resolved in: cf7bafeff0a8b17a5ca721e3ec303f575cf4ca12